.travis.yml

```diff
@@ -27,6 +28,10 @@ before_install:
   - gem update --system
   - gem update bundler
 
+before_script:
+  # Patch for Ruby 2.4.0 compatibility.
+  - ./scripts/patch_simplecov_0_12_0
+
```

Comment:
@nodo yes of course using `simplecov 0.13` is better if we can use it on Travis.
But right now we can not use it on Travis.
Because we are using latest version `coveralls 0.8.19` in our `cucumber-ruby`.
And it needs `simplecov` 0.12.x.

See `Gemfile.lock` file after `bundle install --path vendor/bundle` in your local environment.

Gemfile.lock
```
...
    coveralls (0.8.19)
      json (>= 1.8, < 3)
      simplecov (~> 0.12.0)
...
```

About this situation, I have reported to `coveralls` upstream github.
So, they may change their dependency to use `simplecov 0.13`.

So the script `./scripts/patch_simplecov_0_12_0` is temporary one to pass the test for Ruby 2.4.0.
After they may update `coveralls`, we can remove this script.